Resources/Prototypes/_Mono/Shipyard/Nfsd/mayhem.yml

```
@@ -0,0 +1,32 @@
+  id: Mayhem
+  parent: BaseVessel
+  name: TSK Mayhem
+  description: A corvette equipped with missiles and a small holding cell.
+  price: 72500
+  category: Medium
+  group: Security
+  access: Security
+  shuttlePath: /Maps/_Mono/Shuttles/Nfsd/mayhem.yml
+  guidebookPage: Null
+  class:
+  - Pursuit
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)